tests/wait-block/save-image-indirect-unused-args/Earthfile

```diff
@@ -0,0 +1,21 @@
+VERSION --wait-block 0.6
+
+# unused by test; however required for wait-test common.sh
+deps:
+    FROM scratch
+
+a:
+  FROM alpine:3.15
+  RUN echo a > /data
+  SAVE IMAGE earthly/testimg:f8874c9c-50bd-4634-aa5f-c53f26b69b87
+
+indirect:
+  FROM alpine:3.15
+  ARG myarg
+  BUILD +a
+
+test:
+  WAIT
+    BUILD +indirect --myarg=notused
+    BUILD +indirect --myarg=unused
+  END
```

Comment:
What is the expected outcome of this test. From the looks of it, it should result in an error?

Reply:
Without this fix, we get the error:

```
Error: build target: build main: failed to solve: SAVE IMAGE earthly/testimg:f8874c9c-50bd-4634-aa5f-c53f26b69b87 was already declared (none had --platform values)
```